.changeset/thin-cobras-raise.md

```diff
@@ -0,0 +1,19 @@
+---
+"@salt-ds/lab": minor
+---
+
+Add PillNext
+
+Changes from Pill API: `label` prop should be passed in as children
+Removed selectable and closable variants from Pill
```

Comment:
I dont know if we need to be explicit about the changes now its PillNext
showing the examples should be good enough imo

Reply:
Sounds good 👍